packages/devtools_extensions/example/README.md

```diff
@@ -1,46 +1,126 @@
 # Example DevTools Extension
 
-This is an end-to-end example of a DevTools extension, including
-- the parent `package:foo` that provides the DevTools extension to end-user
-applications (`foo/packages/foo`)
-- the end-user application (`app_that_uses_foo`) that depends on `package:foo`,
-which will trigger a load of the `package:foo` DevTools extension when debugging
-this app with DevTools
-- the `package:foo` DevTools extension (`foo/packages/foo_devtools_extension`),
-which is a Flutter web app that will be embedded in DevTools when debugging an
-app the uses `package:foo`
-
-
-This example will show you how to:
-1. Structure your package for optimal extension development and publishing
+This directory contains end-to-end examples of DevTools extensions. Each
+end-to-end example is made up of three components:
+
+1. **The Dart packages that provide a DevTools extension** to end-user
+applications
+
+    - `package:foo` from `packages_with_extensions/foo/packages/foo`, which is a
+  package that can be used in a Flutter app
+
+    - `package:dart_foo` from `packages_with_extensions/dart_foo/packages/dart_foo`,
+  which is a pure Dart package that can be used in a Dart or Flutter app
+
```

Comment:
which is a pure Dart package that can be used in a Dart or Flutter app - for Dart or Flutter apps

Reply:
shortened